ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -66,6 +66,75 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
         )
   }
 
+  @Explanation(
+    "This error happens when the combined use of all objects in a JVM heap pool exceed the configured limit."
```

Comment:
```suggestion
    "This error happens when the combined use of all objects in a JVM heap pool exceed the configured limit."
```suggestion
    "This error happens when the JVM heap memory pool exceeds a pre-configured limit."
```
I would keep it simpler.